curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody/iterations-more-patterns.md

```diff
@@ -3,6 +3,10 @@ id: 5e7b9f070b6c005b0e76f05f
 title: 'Iterations: More Patterns'
 challengeType: 11
 videoId: 9Wtqo6vha1M
+bilibiliIds:
+  aid: 334491369
+  bvid: BV1tw411R7Mm
+  cid: 376530765
```

Comment:
These seem to be the right values for the "05 - Iterations D - Python for Everybody Course" video used in this challenge:

```suggestion
  aid: 674492981
  bvid: BV1hU4y1H7tF
  cid: 376531204
```

Here's the `iframe` for quick testing:

```
<iframe src="//player.bilibili.com/player.html?aid=674492981&bvid=BV1hU4y1H7tF&cid=376531204&page=1" scrolling="no" border="0" frameborder="no" framespacing="0" allowfullscreen="true"> </iframe>
```

Reply:
Got it. Thank Kris for finding the correct ones.

Reply:
Perfect, thanks @miyaliu666 and @scissorsneedfoodtoo!